src/backend/controllers/VideoController.js

```diff
@@ -51,3 +51,21 @@ export const getVideoHandler = function (schema, request) {
     );
   }
 };
+
+export const getVideoByTypeHandler = function (schema, request) {
```

Comment:
getting video by type can be done in frontend too, but for large app it's a great plus point if we could do so with backend, nice work